docs/animations.md

```diff
@@ -0,0 +1,19 @@
+# Animations
+
+A picture is worth a thousand words. In this spirit, you can use WEBKNOSSOS to create eye-catching animation of your datasets as a video clip. You can use these short movies as part of a presentation, website, for social media or to promote a publication.
```

Comment:
```suggestion
A picture is worth more than a thousand words. In this spirit, you can use WEBKNOSSOS to create an eye-catching animation of your datasets as a video clip. You can use these short movies as part of a presentation, website, for social media or to promote a publication.
```

Reply:
"more than" is Genglish: https://en.wikipedia.org/wiki/A_picture_is_worth_a_thousand_words